### PR TITLE
Correct Pilgrim's Token price

### PR DESCRIPTION
**Pull request by anaseskyrider**
Item from the feat is a wooden religious symbol, so its price should be 1 sp instead of 2 sp.

### CODE REVIEW

**Comment by simonward** [comment]
The price change looks good.

It also looks like the description and publication should be updated to PC2. Would you prefer to add that this this PR, or should I submit a follow-up PR?

**Comment by anaseskyrider** [comment]
The item previously copied the text of the feat with a very slight wording change to fit an item description: "You carry a small token of[...]" -> "A small token of[...]".

I did something similar for this update. I removed the sentence, "You gain a wooden religious symbol for your deity for free.", and referred to the item itself as that wooden symbol: "This wooden religious symbol is a small token of[...]"

File: packs/equipment/pilgrims-token.json [hunk 1]
```diff
@@ -25,13 +25,13 @@
         },
         "price": {
             "value": {
-                "sp": 2
+                "sp": 1
             }
         },
         "publication": {
             "license": "OGL",
```
Comment (simonward) [inline packs/equipment/pilgrims-token.json:32]:
```suggestion
            "license": "ORC",
```

File: packs/equipment/pilgrims-token.json [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
+            "value": "<p>This wooden religious symbol is a small token of protection from a site holy to your faith. As long as this religious symbol is in your possession, when you tie an adversary's initiative roll, you go first.</p><p>If you lose this religious symbol, you must purchase or Craft a replacement and attune it. Such a token usually costs at least 1 sp, and the attunement takes 10 minutes of prayer. You can also attune a different religious symbol with the same amount of time, but you lose the benefit of the previous religious symbol.</p>"
```
Comment (simonward) [inline packs/equipment/pilgrims-token.json:12]:
```suggestion
            "value": "<p>This wooden religious symbol is a small token of protection from a site holy to your faith. As long as this religious symbol is in your possession, when you tie an adversary's initiative roll, you go first.</p>\n<p>If you lose this religious symbol, you must purchase or Craft a replacement and attune it. Such a token usually costs at least 1 sp, and the attunement takes 10 minutes of prayer. You can also attune a different religious symbol with the same amount of time, but you lose the benefit of the previous religious symbol.</p>"
```

**Pull request merged**